kubernetes-core.el

```diff
@@ -11,6 +11,23 @@
 
 (defvar kubernetes-state--current-state nil)
 
+(defmacro kubernetes--save-window-state (&rest body)
```

Comment:
defun?

Reply:
If this is intentional, please adjust the commit log and PR description.

Reply:
What do you mean? I just moved the macro definition to the top of the file. Defining the macro as a function did not work.

Reply:
I wonder how moving around the macro makes the issue go away.

Reply:
https://www.gnu.org/software/emacs/manual/html_node/elisp/Compiling-Macros.html

> In order for compilation of macro calls to work, the macros must already be defined in Lisp when the calls to them are compiled. 

Macro was being called before it is defined.